borsh/src/de/mod.rs

```diff
@@ -908,6 +908,47 @@ impl<T: ?Sized> BorshDeserialize for PhantomData<T> {
         Ok(PhantomData)
     }
 }
+
```

Comment:
add a test file `borsh/tests/test_cells.rs` with roundtrip tests , and `de_errors` and `schema` modules (with at least 1 test each), similar to https://github.com/near/borsh-rs/blob/master/borsh/tests/test_ascii_strings.rs